build.sbt

```diff
@@ -154,6 +153,14 @@ Compile / doc / scalacOptions ++= Seq(
 )
 
 assembly / assemblyJarName := s"${name.value}-assembly-${sparkVerStr.value}_${scalaBinaryVersion.value}-${version.value}.jar"
+assemblyPackageScala / assembleArtifact := false
```

Comment:
Could you add some comment lines about what are these lines doing?

Reply:
See https://github.com/sbt/sbt-assembly/tree/v1.2.0#excluding-scala-library-jars
It is documented in the sbt-assembly README. Let me add a comment about it